### PR TITLE
Added transformcontrols and missing fields

### PR DESCRIPTION
**Pull request by Pro**

### CODE REVIEW

**Comment by vvakame** [comment]
@gyohk could you review this PR?

File: threejs/three.d.ts [hunk 1]
```diff
@@ -5673,6 +5673,7 @@ declare module THREE {
 
     export class BoundingBoxHelper extends Mesh {
         constructor(object: Object3D, hex?: number);
+        constructor();
```
Comment (gyohk) [inline threejs/three.d.ts]: the following seems a bit better.

```
constructor(object?: Object3D, hex?: number);
```

File: threejs/three.d.ts [hunk 2]
```diff
@@ -5791,6 +5792,8 @@ declare module THREE {
     export class WireframeHelper extends Line {
         constructor(object: Object3D, hex?: number);
 
+        object: Object3D;
```
Comment (gyohk) [inline threejs/three.d.ts]: WireframeHelper class does not seem to have the 'object' property.

https://github.com/mrdoob/three.js/blob/master/src/extras/helpers/WireframeHelper.js

Comment (Pro) [inline threejs/three.d.ts]: You are right, I messed that up with the constructor definition. Will update the pull request

**Comment by Pro** [comment]
@gyohk Changed according to your suggestions

Should be fine now?!

**Comment by gyohk** [comment]
Looks fine.
Thank you for the modification.

**Comment by vvakame** [comment]
@gyohk thanks!
@Pro thanks mate!

**Pull request merged**